index.js

```diff
@@ -244,6 +244,19 @@ class Mapeo extends events.EventEmitter {
     })
   }
 
+  getDatabaseStatus () {
```

Comment:
To be reliable, I think this needs to be async and wait on `multifeed.ready()`. If it's called prematurely the feed list will be empty or only partially populated.